virtual-setup/rstudio-login.md

```diff
@@ -67,6 +70,13 @@ You can stop your current R session with the orange circular on/off button in th
 <br>
 <br>
 
+RStudio may ask you if you would like to save your current workspace?
+In general, we advise saying `Don't Save` so you can start fresh in your next session (this will save your objects from the R environment to a file and reload them, more on the R environment later).
```

Comment:
I think the parenthetical is more confusing than enlightening here. What we are advising them to do is *not* save. Alternatively, change "this will" to "Choosing 'Save' at this point would"
```suggestion
In general, we advise choosing `Don't Save` so you can start fresh in your next session.
```

Reply:
I kept rewriting these sentences. Because I don’t want to be confusing but also want them to know what it means.